app/views/foreman_tasks/api/tasks/details.json.rabl

```diff
@@ -9,10 +9,13 @@ node(:failed_steps) { @task.input_output_failed_steps }
 node(:running_steps) { @task.input_output_running_steps }
 node(:help) { troubleshooting_info_text }
 node(:has_sub_tasks) { @task.sub_tasks.any? }
+node(:allowDangerousActions) { Setting['dynflow_allow_dangerous_actions'] }
```

Comment:
Why do we use `camelCase` here?

Reply:
This is a bad rebase, it doesn't seem to be used anywhere.